lib/aws-ecs.go

```diff
@@ -122,6 +125,17 @@ func (p ECSPlugin) FetchMetrics() (map[string]float64, error) {
 	stat := make(map[string]float64)
 
 	for name := range p.GraphDefinition() {
+		if name == "Task" {
+			met := metrics{"CPUUtilization", metricsTypeSampleCount}
```

Comment:
Is `CPUUtilization` is right? 

I think `name` seems enough.

Reply:
`ClusterName`, `ServiceName` dimention don't have "Task" metrics, only have `CPUUtilization` and `MemoryUtilization` metrics.
So, I should get `SampleCount` stats from either metrics.

Reply:
Sorry, I misunderstood :bow: